packages/manager/src/features/linodes/LinodesCreate/ApiAwarenessModal/index.test.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import { renderWithTheme } from 'src/utilities/testHelpers';
+
+import ApiAwarenessModal, { Props } from '.';
+
+const defaultProps: Props = {
+  isOpen: false,
+  onClose: jest.fn(),
+};
+
+const renderComponent = (overrideProps?: Pick<Props, 'isOpen'>) => {
```

Comment:
`Pick` may not be the right type here. I think `Partial` makes more sense. `Partial` makes all properties in the Props object optional. 

```suggestion
const renderComponent = (overrideProps?: Partial<Props>) => {
```